docs/rules/newline-after-var.md

```diff
@@ -0,0 +1,39 @@
+# Allow/disallow an empty newline after var statement (newline-after-var)
+
+The `newline-after-var` rule enforces a coding style where empty newlines are allowed or disallowed after `var` statement.
```

Comment:
This section should not be about the rule, but rather about the theory behind the rule. Why do people put a newline after variable statements? What does it look like? See http://eslint.org/docs/rules/no-sparse-arrays.html as an example.